modules/page_object_about_prefs.py

```diff
@@ -323,6 +323,30 @@ def get_iframe(self) -> WebElement:
         """
         return self.get_element("browser-popup")
 
+    def set_alternative_language(self, lang_code: str) -> BasePage:
```

Comment:
i know there is another suite of language pack tests, can they be refactored to use this or is this specific to your tests? (even if it can be refactored thats future work, just wondering)

Reply:
Let me look into that...I thought those tests were looking at preferred content language

Reply:
Alright, yeah one of those tests was refactored!